api/go.mod

```diff
@@ -81,6 +73,121 @@ require (
 	sigs.k8s.io/yaml v1.2.0
 )
 
+require (
```

Comment:
Why is there two require blocks?

Reply:
You might need to run `go mod tidy`

Reply:
It's because in 1.17 they implements https://go-review.googlesource.com/c/go/+/325922/

Reply:
I see. #TIL